lib/galaxy/tool_util/xsd/galaxy.xsd

```diff
@@ -3968,14 +3968,18 @@ dataset for the contained input of the type specified using the ``type`` tag.
   <xs:complexType name="ParamOptions">
     <xs:annotation>
       <xs:documentation xml:lang="en"><![CDATA[
-
 See [/tools/extract/liftOver_wrapper.xml](https://github.com/galaxyproject/galaxy/blob/master/tools/extract/liftOver_wrapper.xml)
 for an example of how to use this tag set. This tag set is optionally contained
 within the ``<param>`` tag when the ``type`` attribute value is ``select`` or
-``data`` and used to dynamically generated lists of options. This tag set
-dynamically creates a list of options whose values can be
-obtained from a predefined file stored locally or a dataset selected from the
-current history.
+``data`` and used to dynamically generated lists of options.
+
+For data parameters it can only be used to restrict inputs to datasets having
+the same ``dbkey`` like another input by using a ``data_meta`` filter. See for
+instance here: [/tools/maf/interval2maf.xml](https://github.com/galaxyproject/galaxy/blob/master/tools/maf/interval2maf.xml)
```

Comment:
How about:

```suggestion
For parameters with ``type="data"`` it can be used to allow only datasets having
the same ``dbkey`` like the input used in another data parameter by using a ``data_meta`` filter. See for
instance here: [/tools/maf/interval2maf.xml](https://github.com/galaxyproject/galaxy/blob/master/tools/maf/interval2maf.xml)
```

Reply:
What I don't understand is:
- what is "it", I assume the tag set ?
- what is "like" referring to, should there be a comma before it ?

Is this capturing what you want to say ?

```suggestion
For data parameters this tag can be used to restrict possible input datasets to datasets that match the ``dbkey`` of another data input that specifies a ``data_meta`` filter. See for
instance here: [/tools/maf/interval2maf.xml](https://github.com/galaxyproject/galaxy/blob/master/tools/maf/interval2maf.xml)
```

but I think you're describing `<options></options` tag, while this described what you can do with filters. If that is true I would not document that here, I think that is the main confusion for me.

Reply:
Thanks for the suggestion. Nearly there .. 

```suggestion
For data parameters this tag can be used to restrict possible input datasets to datasets that match the ``dbkey`` of another data input by including a ``data_meta`` filter. See for
instance here: [/tools/maf/interval2maf.xml](https://github.com/galaxyproject/galaxy/blob/master/tools/maf/interval2maf.xml)
```